src/System.Runtime.Intrinsics.X86/ref/System.Runtime.Intrinsics.X86.cs

```diff
@@ -0,0 +1,1118 @@
+        public static Vector256<float> HorizontalAdd(Vector256<float> left, Vector256<float> right) { throw null; }
+        public static Vector256<double> HorizontalAdd(Vector256<double> left, Vector256<double> right) { throw null; }
+        public static Vector256<float> HorizontalSubtract(Vector256<float> left, Vector256<float> right) { throw null; }
+        public static Vector256<double> HorizontalSubtract(Vector256<double> left, Vector256<double> right) { throw null; }
+        public static Vector256<T> InsertSbyte<T>(Vector256<T> value, sbyte data, byte index) where T : struct { throw null; }
+        public static Vector256<T> InsertByte<T>(Vector256<T> value, byte data, byte index) where T : struct { throw null; }
+        public static Vector256<T> InsertShort<T>(Vector256<T> value, short data, byte index) where T : struct { throw null; }
+        public static Vector256<T> InsertUshort<T>(Vector256<T> value, ushort data, byte index) where T : struct { throw null; }
+        public static Vector256<T> InsertInt<T>(Vector256<T> value, int data, byte index) where T : struct { throw null; }
+        public static Vector256<T> InsertUint<T>(Vector256<T> value, uint data, byte index) where T : struct { throw null; }
+        public static Vector256<T> InsertLong<T>(Vector256<T> value, long data, byte index) where T : struct { throw null; }
+        public static Vector256<T> InsertUlong<T>(Vector256<T> value, ulong data, byte index) where T : struct { throw null; }
+        public static Vector256<T> Insert<T>(Vector256<T> value, Vector128<T> data, byte index) where T : struct { throw null; }
+        public static unsafe Vector256<sbyte> Insert(Vector256<sbyte> value, sbyte* address, byte index) { throw null; }
+        public static unsafe Vector256<byte> Insert(Vector256<byte> value, byte* address, byte index) { throw null; }
+        public static unsafe Vector256<short> Insert(Vector256<short> value, short* address, byte index) { throw null; }
+        public static unsafe Vector256<ushort> Insert(Vector256<ushort> value, ushort* address, byte index) { throw null; }
+        public static unsafe Vector256<int> Insert(Vector256<int> value, int* address, byte index) { throw null; }
+        public static unsafe Vector256<uint> Insert(Vector256<uint> value, uint* address, byte index) { throw null; }
+        public static unsafe Vector256<long> Insert(Vector256<long> value, long* address, byte index) { throw null; }
+        public static unsafe Vector256<ulong> Insert(Vector256<ulong> value, ulong* address, byte index) { throw null; }
+        public static unsafe Vector256<float> Insert(Vector256<float> value, float* address, byte index) { throw null; }
+        public static unsafe Vector256<double> Insert(Vector256<double> value, double* address, byte index) { throw null; }
+        public static unsafe Vector256<sbyte> Load(sbyte* address) { throw null; }
```

Comment:
💭 Semantically, I believe the proposal would benefit tremendously from using `ref Vector256<sbyte>` here. This applies to many of the methods involving load/store operations for contiguous memory locations.